### PR TITLE
Add r-abcrf 1.9

### PR DESCRIPTION
**Pull request by G-Sarah**
# Add recipe for r-abcrf 1.9

This PR adds the 1.9 release of the [abcrf R package](https://cran.r-project.org/package=abcrf) (Approximate Bayesian Computation via Random Forests) to Conda Forge. The recipe was generated using `conda skeleton cran r-abcrf`. 

Pudlo P., Marin J.-M., Estoup A., Cornuet J.-M., Gautier M. and Robert C. P. (2016) <[doi:10.1093/bioinformatics/btv684](https://doi.org/10.1093%2Fbioinformatics%2Fbtv684)>.
Estoup A., Raynal L., Verdu P. and Marin J.-M. <http://journal-sfds.fr/article/view/709>.
Raynal L., Marin J.-M., Pudlo P., Ribatet M., Robert C. P. and Estoup A. (2019) <[doi:10.1093/bioinformatics/bty867](https://doi.org/10.1093%2Fbioinformatics%2Fbty867)>.

-----

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by G-Sarah** [comment]
Hello @cbrueffer, I'm tagging you liked you asked on my PR over at bioconda since the checks are done running here. The winhdows test has failed though, and I'm not sure what to do about it. Would you by any chance have tips for resolving this issue?

Thank you for your help!

File: recipes/r-abcrf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    - r-doparallel
+    - r-foreach
+    - r-matrixstats
+    - r-ranger
+    - r-readr
+    - r-stringr
+
+test:
+  commands:
+    # You can put additional test commands to be run here.
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:66]: Remove unspecific comments.

```suggestion
```

File: recipes/r-abcrf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    - $R -e "library('abcrf')"           # [not win]
+    - "\"%R%\" -e \"library('abcrf')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
+  # in the recipe that will be run at test time.
+
+  # requires:
+    # Put any additional test requirements here.
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:74]: Remove unspecific comments.

```suggestion
```

File: recipes/r-abcrf/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# NeedsCompilation: yes
+# Maintainer: Jean-Michel Marin <jean-michel.marin@umontpellier.fr>
+# Repository: CRAN
+# Author: Jean-Michel Marin [aut, cre], Louis Raynal [aut], Pierre Pudlo [aut], Christian P. Robert [ctb], Arnaud Estoup [ctb]
+# Packaged: 2022-08-09 09:16:29 UTC; jmm
+# Date/Publication: 2022-08-09 15:30:19 UTC
+
+# See
+# https://docs.conda.io/projects/conda-build for
+# more information about meta.yaml
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:115]: Remove unspecific comments.

```suggestion
```

File: recipes/r-abcrf/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  home: https://CRAN.R-project.org/package=abcrf
+  license: GPL-2.0-or-later
+  summary: Performs Approximate Bayesian Computation (ABC) model choice and parameter inference
+    via random forests. Pudlo P., Marin J.-M., Estoup A., Cornuet J.-M., Gautier M.
+    and Robert C. P. (2016) <doi:10.1093/bioinformatics/btv684>. Estoup A., Raynal L.,
+    Verdu P. and Marin J.-M. <http://journal-sfds.fr/article/view/709>. Raynal L., Marin
+    J.-M., Pudlo P., Ribatet M., Robert C. P. and Estoup A. (2019) <doi:10.1093/bioinformatics/bty867>.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:86]: Include alternative license:

```suggestion
     - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
     - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3
```

File: recipes/r-abcrf/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:26]: Workaround for Windows error (see https://github.com/conda/conda-build/pull/4786):

```suggestion
  missing_dso_whitelist:
     - '*/R.dll'            # [win]

```

File: recipes/r-abcrf/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+    Verdu P. and Marin J.-M. <http://journal-sfds.fr/article/view/709>. Raynal L., Marin
+    J.-M., Pudlo P., Ribatet M., Robert C. P. and Estoup A. (2019) <doi:10.1093/bioinformatics/bty867>.
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - G-Sarah
+
```
Comment (mfansler) [inline recipes/r-abcrf/meta.yaml:91]: Include R team.

```suggestion
    - conda-forge/r

```

**Comment by G-Sarah** [comment]
> Thank you for contributing this! A few changes to merge, but mostly there.
> 
> In the future, we prefer contributors to use [`conda_r_skeleton_helper`](https://github.com/bgruening/conda_r_skeleton_helper) to generate Conda Forge recipes for CRAN packages.

Thank you!

**Review by mfansler — APPROVED** [review]

**Comment by cbrueffer** [comment]
Thanks for picking this up @mfansler !

**Pull request merged**